tests/tests_fabric/parity/test_parity_ddp.py

```diff
@@ -125,7 +125,6 @@ def train_fabric_ddp(fabric):
     return model.state_dict(), torch.tensor(iteration_timings), memory_stats
 
 
-@pytest.mark.flaky(reruns=3)
```

Comment:
This dependency is now unused, no? I believe you added it only for this

Reply:
It is still used in another test. I would like to keep it. We may need it again if the machine gets noisy.